### PR TITLE
Feature cob4 1 without arms

### PR DESCRIPTION
**Pull request by fmessmer**
builds on top of #368 (respective commits will be removed after #368 is merged)

As discussed: 
For our new project, we use cob4-2 without arms. In order to simplify/separate configuration, we use cob4-1 in the required setup (cob4-1 was planned to be removed anyway).
Therefore, I meld'ed and merged the latest cob4-2 setup into cob4-1 setup and removed everything related to head, arms and gripper in a single commit (https://github.com/ipa320/cob_robots/commit/c1f2bf6cbb4403da798c37b11bed9baea6c6424d)

requires: https://github.com/ipa320/cob_calibration_data/pull/93

### CODE REVIEW

File: cob_bringup/robots/cob4-1.launch [hunk 1]
```diff
@@ -5,17 +5,17 @@
...
-		<arg name="cob4-1-t2" value="cob4-1-t2"/>
-		<arg name="cob4-1-t3" value="cob4-1-t3"/>
-		<arg name="cob4-1-s1" value="cob4-1-s1"/>
-		<arg name="cob4-1-h1" value="cob4-1-h1"/>
+		<arg name="cob4-1-b1" value="cob4-2-b1"/>
+		<arg name="cob4-1-t1" value="cob4-2-t1"/>
+		<arg name="cob4-1-t2" value="cob4-2-t2"/>
+		<arg name="cob4-1-t3" value="cob4-2-t3"/>
+		<arg name="cob4-1-s1" value="cob4-2-s1"/>
+		<arg name="cob4-1-h1" value="cob4-2-h1"/>
```
Comment (fmessmer) [inline cob_bringup/robots/cob4-1.launch:13]: this is required as we run the cob4-1 setup on the cob4-2 hardware and the PCs are named like that

File: cob_bringup/robots/cob4-1.launch [hunk 2]
```diff
@@ -5,17 +5,17 @@
 	<arg name="pkg_env_config" default="$(find cob_default_env_config)"/>
```
Comment (NadiaHG) [inline cob_bringup/robots/cob4-1.launch:5]: Please comment out also the arguments,  if you don' t define a ROBOT_ENV variable you will get an error and you are not using it...

File: cob_hardware_config/cob4-1/config/sensorring_driver.yaml [hunk 3]
```diff
@@ -2,9 +2,11 @@ name: sensorring_controller
 
 defaults:
   eds_pkg: cob_hardware_config
-  eds_file: "cob4-1/config/Elmo.dcf"
+  eds_file: "cob4-2/config/Elmo.dcf"
```
Comment (NadiaHG) [inline cob_hardware_config/cob4-1/config/sensorring_driver.yaml]: why cob4-2? is it different? the Elmo.dcf should be a common file , and the cobX-Y/config/Elmo.dcf just a link

File: cob_hardware_config/cob4-1/config/torso_driver.yaml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 defaults:
   eds_pkg: cob_hardware_config
-  eds_file: "cob4-1/config/Elmo.dcf"
+  eds_file: "cob4-2/config/Elmo.dcf"
```
Comment (NadiaHG) [inline cob_hardware_config/cob4-1/config/torso_driver.yaml]: why cob4-2? is it different? the Elmo.dcf should be a common file , and the cobX-Y/config/Elmo.dcf just a link

File: cob_hardware_config/cob4-1/config/sensorring_driver.yaml (hunk 3 above)
Comment (fmessmer) [inline cob_hardware_config/cob4-1/config/sensorring_driver.yaml]: Well spotted...thx

File: cob_hardware_config/cob4-1/config/torso_driver.yaml (hunk 4 above)
Comment (fmessmer) [inline cob_hardware_config/cob4-1/config/torso_driver.yaml]: Well spotted...thx

File: cob_bringup/robots/cob4-1.launch (hunk 2 above)
Comment (fmessmer) [inline cob_bringup/robots/cob4-1.launch:5]: Good point, this is something more general, though.
The point of commenting out the `upload_navigation_goals.launch` below was copied from cob4-2.
By doing so, indeed the launch args become obsolete (for all robots)!

But if we uncomment the `upload_navigation_goals.launch` here, where else is it loaded?
Should be somewhere within navigation, right?
We just should make sure that navigation does not rely on navigation goals to be uploaded via robot.launch but take care about it itself....
@ipa-mig @ipa-srd @ipa-flg  do you have a comment here?

Comment (mgruhler) [inline cob_bringup/robots/cob4-1.launch:5]: Well, the navigation does not rely on this file at all, afaik.
This file is used by the dashboard to specify the goals, which are then sent via scriptserver to the navigation.
I don't know of any other case, but maybe it is good to listen to @ipa-srd and @ipa-flg as well.

Comment (stefandoerr) [inline cob_bringup/robots/cob4-1.launch:5]: agree with @ipa-mig

Comment (fmessmer) [inline cob_bringup/robots/cob4-1.launch:5]: Will be handled when dealing with #373

**Comment by floweisshardt** [comment]
FR: OK

**Comment by fmessmer** [comment]
Tested on cob4-1 aka. cob4-2 without arms (dashboard, joy motion, twist_control, lookat, base_active)

**Comment by floweisshardt** [comment]
FR: Ok

**Pull request merged**